doc/en/components/grids/_shared/sorting.md

```diff
@@ -0,0 +1,365 @@
+---
+title: {Platform} {ComponentTitle} Sorting - {ProductName}
+_description: Get started with the {Platform} sorting feature of {ProductName} {ComponentTitle}! Configure a mix of sortable columns & change the display order of data records.
+_keywords: {Platform} sort, {Platform}, {ProductName}, Infragistics
+mentionedTypes: [{ComponentApiMembers}]
+sharedComponents: ["Grid", "TreeGrid", "HierarchicalGrid"]
+---
+
+# {Platform} {ComponentTitle} Sorting
+
+In {Platform} {ComponentTitle}, data sorting is enabled on a per-column level, meaning that the **{ComponentSelector}** can have a mix of sortable and non-sortable columns. Performing angular sort actions enables you to change the display order of the records based on specified criteria.
+
+
+> Note: Up until now, grouping/sorting worked in conjuction with each other. In 13.2 version, a new behavior which decouples gropuing from sorting is introduced. For example - clearing the grouping will not clear sorting expressions in the grid or vice versa. Still, if a column is both sorted and grouped, grouped expressions take precedence.
+
+## {Platform} {ComponentTitle} Sorting Overview Example
+
+<!-- ComponentStart: Grid -->
+
+<code-view style="height:550px"
+           data-demos-base-url="{environment:demosBaseUrl}"
+           iframe-src="{environment:demosBaseUrl}/{ComponentSample}-sorting-sample"
+           github-src="{ComponentSample}/sorting-sample" >
+</code-view>
+
+<!-- ComponentEnd: Grid -->
+
+
+<!-- ComponentStart: TreeGrid -->
+
+Additionally there is a custom contextmenu added for sorting using **{ComponentSelector}**'s `ContextMenu` Output.
+
+<code-view style="height:550px"
+           data-demos-base-url="{environment:demosBaseUrl}"
+           iframe-src="{environment:demosBaseUrl}/{ComponentSample}-sorting"
+           github-src="{ComponentSample}/sorting" >
+</code-view>
+
+<!-- ComponentEnd: TreeGrid -->
+
+<!-- ComponentStart: HierarchicalGrid -->
+
+Additionally there is a custom contextmenu added for sorting using **{ComponentSelector}**'s `ContextMenu` Output.
+
+<code-view style="height:510px"
+           data-demos-base-url="{environment:demosBaseUrl}"
+           iframe-src="{environment:demosBaseUrl}/{ComponentSample}-sorting"
+           github-src="{ComponentSample}/sorting" >
+
+</code-view>
```

Comment:
replace above 3 code-view blocks with just one code-view and remove component build flag because routing path to the sample will be the same for all grid samples in Sample Browser.